pheps/phep-9999.md

```diff
@@ -0,0 +1,112 @@
+
+| Tier                  | Summer School Inclusion | PyHC Software Env Inclusion | PyHC-Chat Bot Inclusion | pyOpenSci Verified badge | Standards Compliance Assistance | Listing on Main PyHC Project Page | Listing on Secondary PyHC Project Page | Software Search Interface Inclusion | Consideration for Conference Travel Funding |
+| :--:                  | :---------------------: | :-------------------------: | :---------------------: | :----------------------: | :-----------------------------: | :-------------------------------: | :------------------------------------: | :---------------------------------: | :-----------------------------------------: |
+| **Gold**              | Yes                     | Yes                         | Yes                     | Yes                      | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Bronze**            | No                      | No                          | No                      | No                       | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Silver**            | No                      | No                          | No                      | No                       | No                              | Yes                               | No                                     | Yes                                 | No                                          |
+| **Honorable Mention** | No                      | No                          | No                      | No                       | No                              | No                                | Yes                                    | Yes                                 | No                                          |
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC Software Env Inclusion: indicates whether a package will be included within the PyHC software environment
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
+   - pyOpenSci Verified Badge: a badge that shows whether a package has completed the pyOpenSci review process
+   - Standards Compliance Assistance: indicates whether a package will receive extra help and/or advice from PyHC leadership in conforming to the PyHC standards
+   - Listing on Main PyHC Project Page: indictes whether a package's information will be displayed on a new, main PyHC Project page
+   - Listing on Secondary PyHC Project Page: indicates whetheer a package's information will be displayed on a new, secondary PyHC Project page 
```

Comment:
Would also be nice to just have a front page with a search bar that could let people find any project by name or by keyword.

Reply:
We technically have this with @sapols 's updates to the Project page, but it could be made more clear, or placed in a more obvious location?